src/modules/teams/containers/TeamsPage.js

```diff
@@ -0,0 +1,99 @@
+import { h, Component } from "preact";
+import { connect } from "preact-redux";
+import { Page } from "src/components/Page";
+import { Team } from "../components/Team";
+
+import styles from "./TeamsPage.scss";
+
+import { fetchAllTeams, replyToInvite, selectTeam } from "../actions";
+import { loadTeams } from "../../../localStorage";
+
+import { route } from "preact-router";
+import I18n from "src/config/i18n";
+
+export class TeamsPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.props.fetchAllTeams(this.props.user.apiToken);
+    }
+
+    componentWillMount() {
+        if (this.props.team.id != undefined) {
+            route("/", true);
+        }
+    }
+
+    render() {
+        console.log(this.props);
+        return (
+            <Page>
+                <main class={styles.main}>
+                    {/* When no teams or invites are available to show. */}
+                    {this.props.amountOfInvites == 0 && this.props.amountOfTeams == 0 ? (
+                        <div className={styles.noTeams}>{I18n.t("teams.no_teams")}</div>
+                    ) : (
+                            <div></div>
```

Comment:
Why show an empty div? You can even omit the whole else statement here if you don't want to return anything.